noisereduction/TrackUtils.cpp

```diff
@@ -28,10 +29,12 @@ InputTrack TrackUtils::readOneTrackFromContext(const SndContext &ctx, int channe
     sf_seek(ctx.file, t0, SEEK_SET);
     float* writePtr = &buffer[0];
 
+    // allocating on heap because variable size stack arrays are evil!
+    float* frameBuffer = new float[ctx.info.channels];
```

Comment:
its too bad VLAs aren't supported on MSVC :(

Can you stick it in `auto_ptr` instead of doing the explicit `delete`?

Reply:
Well, it is not part of the C++ standard but of C99, so I think it is actually good that it is forbidden :D Yeah I can do that, or just use 'std::vector', but I guess it is overkill because the size of the array is fixed.